os/net/netmgr/netdev_lwip.c

```diff
@@ -372,9 +372,15 @@ static int lwip_input(struct netdev *dev, void *frame_ptr, uint16_t len)
 	LWIP_DEBUGF(NETIF_DEBUG, ("processing pbufs\n"));
 
 	/* We iterate over the pbuf chain until we have read the entire packet into the pbuf. */
+	uint32_t total = 0;
+	uint32_t remain = len;
 	for (q = p; q != NULL; q = q->next) {
-		memcpy(q->payload, frame_ptr, q->len);
-		frame_ptr += q->len;
+		int buf_size = q->len > remain ? remain : q->len;
```

Comment:
In pbuf_ alloc, there is
>p->len = LWIP_MIN(length, PBUF_POOL_BUFSIZE_ALIGNED - LWIP_MEM_ALIGN_SIZE(offset));
In my opinion. q->len is always not not more than remain.

Reply:
ok. it seems unnecessary code. I'll not apply it to public.